osbrain/tests/test_common.py

```diff
@@ -30,3 +31,36 @@ def test_loglevel():
     # Invalid initialization
     with pytest.raises(ValueError):
         LogLevel('FOO')
+
+
+def test_topics_to_bytes():
+    """
+    Test topics_to_bytes basic usage and required types.
+    """
+    handlers = {'topic_one': 'handler', b'topic_two': 'handler'}
+
+    # Valid call (without uuid)
+    curated = topics_to_bytes(handlers)
+    assert isinstance(curated, dict)
+
+    for k, v in curated.items():
+        assert isinstance(k, bytes)
+        assert k.startswith(b'topic_')
+        assert v == 'handler'
+
+    # Valid call (with uuid)
+    curated = topics_to_bytes(handlers, uuid=b'U')
+    assert isinstance(curated, dict)
+
+    for k, v in curated.items():
+        assert isinstance(k, bytes)
+        assert k.startswith(b'Utopic_')
+        assert v == 'handler'
+
+    # Wrong parameter (must be a dictionary)
+    with pytest.raises(AttributeError):
+        topics_to_bytes([1, 2])
+
+    # Wrong topic types (must be `str` or `bytes`)
+    with pytest.raises(TypeError):
+        topics_to_bytes({1: 2})
```

Comment:
Hehehe. These two `pytest.raises` do still work, but I think we should remove them from the test:

- `AttributeError`: raises because a list does not have `.items()` method.
- `TypeError`: raises because you cannot concat bytes to int.